.travis.yml

```diff
@@ -4,9 +4,14 @@ cache: bundler
 rvm:
 - 2.4.5
 - 2.5.3
+env:
+- TEST_BUNDLER_VERSION=1.15
+- TEST_BUNDLER_VERSION=1.16
+- TEST_BUNDLER_VERSION=1.17
+- TEST_BUNDLER_VERSION=2.0
```

Comment:
I couldn't use BUNDLER_VERSION, because bundler itself uses that internally (it sets it after it reads the lockfile, or you can specify to override the lockfile), so instead I went with this more explicit env var name.